packages/@orbit/data/src/exception.ts

```diff
@@ -104,6 +104,34 @@ export class TransformNotAllowed extends Exception {
 }
 
 /**
+ * An error occured related to the schema.
+ * 
+ * @export
+ * @class SchemaError
+ */
+export class SchemaError extends Exception {
+  public description: string;
+
+  constructor(description: string) {
```

Comment:
I've been favoring separate property definitions for clarity of documentation. I'm open to seeing alternative examples though :)